### PR TITLE
Fixed build and test of rc2 branch in CI

### PR DESCRIPTION
**Pull request by wtgodbe**
This should fix the failing rc2 test legs in the CI

### CODE REVIEW

**Comment by wtgodbe** [comment]
@mmitche how does this look?

**Comment by sergiy-k** [comment]
@dotnet-bot please test this

**Comment by wtgodbe** [comment]
@dotnet-bot test this please

**Comment by mmitche** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -422,13 +433,7 @@ combinedScenarios.each { scenario ->
...
 
                     // Add all the standard options
-                    def branchParam = ''
-                    if (branchName == 'release/1.0.0-rc2'){
-                        branchParam = '*/${branchName}'
-                    } else {
-                        branchParam = '*/master'
-                    }
-                    Utilities.standardJobSetup(newJob, project, isPR, branchParam)
+                    Utilities.standardJobSetup(newJob, project, isPR, getFullBranchName(branch))
```
Comment (mmitche) [inline netci.groovy]: branchName

**Comment by wtgodbe** [comment]
@dotnet-bot test this please

**Comment by wtgodbe** [comment]
@dotnet-bot test ci please

File: netci.groovy (hunk 1 above)
Comment (wtgodbe) [inline netci.groovy]: Whoops, just fixed these

**Comment by mmitche** [comment]
@wtgodbe You don't need to test this pleas on this (in fact all the builds will just skip).  Just the ci test phrase.

**Comment by wtgodbe** [comment]
@mmitche gotcha. Looks like generator_prtest passed, does all else look good to you?

**Comment by mmitche** [comment]
LGTM merge away.

**Pull request merged**